<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Conversor de Horas a Segundos</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="lbl_ingresar_hora">
      <property name="text">
       <string>Ingrese la hora (HH:MM:SS):</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="txt_hora"/>
    </item>
    <item>
     <widget class="QPushButton" name="btn_calcular">
      <property name="text">
       <string>Calcular</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lbl_resultado">
      <property name="text">
       <string>Segundos transcurridos:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="txt_segundos">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_reiniciar">
      <property name="text">
       <string>Reiniciar</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_salir">
      <property name="text">
       <string>Salir</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <resources/>
  <connections/>
 </widget>
</ui>
